### ISSUE TITLE
Problem with page refresh

### ISSUE THREAD

**Issue by visavi** [marxjmoura/admin4b#74]
If you refresh the page from anywhere F5 or ctrl+R, it always flips up, is there any way to fix this?
Chrome and firefox desktop

**Comment by marxjmoura** [comment]
You are right. These browsers remember the scroll position of the `<body>` tag. Admin4B hide the overflow content of body tag and uses `.app .app-content .content-body` as the body, which is causing this user experience.

**Comment by marxjmoura** [comment]
Could you try this for now?

```js
window.onbeforeunload = function () {
  localStorage.setItem('scrollTop', $('.content-body').scrollTop());
};

$('.content-body').scrollTop(localStorage.scrollTop || 0);
```

**Comment by visavi** [comment]
yes it works, it's just that it’s bad on js

problem
html, body {
   height: 100%;
   width: 100%}

if you remove the height, then everything is fine, but the left panel does not work

**Comment by marxjmoura** [comment]
I know, it is not the better solution. :/

Maybe the real solution involves changing the CSS hierarchy. I will work on this in the next couple of weeks.

**Comment by visavi** [comment]
Thank you, I will wait for changes
